pkg/render/manager.go

```diff
@@ -302,20 +303,25 @@ func (c *managerComponent) managerVolumes() []v1.Volume {
 		v = append(v,
 			v1.Volume{
 				// We only want to mount the cert, not the private key to es-proxy to establish a connection with voltron.
-				Name: ManagerTLSSecretCertName,
+				Name: ManagerInternalTLSSecretName,
 				VolumeSource: v1.VolumeSource{
 					Secret: &v1.SecretVolumeSource{
```

Comment:
Does this mean you are also mounting a private key to es-proxy now, unlike what the comment says?

Reply:
Talked offline: We can declare the volume twice: a declaration with cert and key for Voltron and a declaration with only the cert for es proxy.